docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/LoadFromEnumerable.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+
+namespace Samples.Dynamic
+{
+    public static class LoadFromEnumerable
+    {
+        // A simple case of creating IDataView from IEnumerable.
+        public static void Example()
+        {
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Get a small dataset as an IEnumerable.
+            IEnumerable<DataPointVector> enumerableKnownSize = new DataPointVector[]
+            {
+               new DataPointVector{ Features = new float[]{ 1.2f, 3.4f, 4.5f, 3.2f, 7,5f } },
+               new DataPointVector{ Features = new float[]{ 4.2f, 3.4f, 14.65f, 3.2f, 3,5f } },
+               new DataPointVector{ Features = new float[]{ 1.6f, 3.5f, 4.5f, 6.2f, 3,5f } },
+            };
+
+            // Load dataset into an IDataView. 
+            IDataView data = mlContext.Data.LoadFromEnumerable(enumerableKnownSize);
+            var featureColumn = data.Schema["Features"].Type as VectorDataViewType;
+            // Inspecting the schema
+            Console.WriteLine($"Is the size of the Features column known: {featureColumn.IsKnownSize}.\nSize: {featureColumn.Size}");
+
+            // Preview
+            //
+            // Is the size of the Features column known? True.
+            // Size: 5.
+
+            // If the size of the vector is unknown at compile time, it can be set at runtime.
+            IEnumerable<DataPoint> enumerableUnknownSize = new DataPoint[]
+            {
+               new DataPoint{ Features = new float[]{ 1.2f, 3.4f, 4.5f } },
+               new DataPoint{ Features = new float[]{ 4.2f, 3.4f, 1.6f } },
+               new DataPoint{ Features = new float[]{ 1.6f, 3.5f, 4.5f } },
+            };
+
+            // The feature dimension retrievable at runtime.
+            int featureDimension = 3;
```

Comment:
is this a misplaced comment ?  .. featureDimension is not used until line 61   for setting , not retrieving ..

Reply:
The sample is for cases when we don't know the size at compile time, so this is is set to a constant, but in real life would be the size of some array. 

But good point, i don't have all that language there, and i don't think it is necessary; so I'll remove the comment :)

---
In reply to: [283514291](https://github.com/dotnet/machinelearning/pull/3696#discussion_r283514291) [](ancestors = 283514291)